features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/ApplicationPolicies.js

```diff
@@ -91,36 +106,26 @@ export default class ApplicationPolicies extends Component {
 
         return (
             <div>
-                <AppBar position="static" >
-                    <Toolbar style={{minHeight:'30px'}}>
-                        <IconButton color="contrast" aria-label="Menu">
-                            <MenuIcon />
-                        </IconButton>
-                        <Link to={"/policies/application_policies/create/"}>
-                             <Button color="contrast">Add Policy</Button>
-                        </Link>
-                    </Toolbar>
-                </AppBar>
+               
                 <Message ref={a => this.msg = a}/>
-                <Grid container>
+                <Grid container justify="center" alignItems="center">
                     <Grid item xs={12}>
-                        <Paper>
-                            <Typography className="page-title" type="display1" gutterBottom>
-                               Application Rate Limits
+                    
+                        <div className={styles.titleWrapper}>
+                            <Typography variant="display1" gutterBottom >
+                                    Application Rate Limits
                             </Typography>
+                            <Link to={"/policies/application_policies/create"} className={styles.createButton}>
+                                <Button variant="raised" color="primary" className={styles.button}>
+                                Add Application Policy
+                                </Button>
+                            </Link>
                             <Typography type="caption" gutterBottom align="left" className="page-title-help">
                             Discription goes here.
-                            </Typography>
-
-                            <Divider />
-                            <div className="page-content">
-                              <TextField
-                                label="Search"
-                                margin="normal"
-                              />
-                            </div>
-                            <Divider />
-                        </Paper>
+                            </Typography>                          
+                        </div>
+                        <Divider className={styles.divider} />
```

Comment:
you need to get class name from classes property, please refer [overriding-with-classes](https://material-ui-next.com/customization/overrides/#overriding-with-classes) doc